cli/cmd/install_test.go

```diff
@@ -318,25 +318,31 @@ metadata:
 }
 
 func TestRenderCRDsWithMissingGatewayAPI(t *testing.T) {
-
 	k, err := k8s.NewFakeAPIFromManifests([]io.Reader{})
 	if err != nil {
 		t.Fatalf("failed to initialize fake API: %s", err)
 	}
 
 	var buf bytes.Buffer
-	err = renderCRDs(context.Background(), k, &buf, values.Options{}, "yaml")
+	err = renderCRDs(context.Background(), k, &buf, values.Options{
+		Values: []string{"installGatewayAPI=false"},
+	}, "yaml")
```

Comment:
I think this test should pass without this change.  The case this tests is where the Gateway API is not already present on the cluster and the user installs Linked with no options.  In this case, the desired behavior is that Linkerd prompts the user to install the Gateway API as a prerequisite before installing Linkerd.